### PR TITLE
Add contributing guide

### PR DESCRIPTION
**Pull request by ManaSugi**
Add contributing guide for libseccomp-rs project.

Fixes: #92

Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Comment by ManaSugi** [comment]
@rusty-snake welcome any suggestions you want to add:)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+[new-issue]: https://github.com/libseccomp-rs/libseccomp-rs/issues/new
+[upstream-libseccomp]: https://github.com/seccomp/libseccomp
+
+### Feature / API Requests
+
+If you'd like a new API or feature added, please open a new issue requesting it.
+As with reporting a bug, the more information you can provide, the better.
+
+## Pull Requests
```
Comment (rusty-snake) [inline CONTRIBUTING.md:40]: When we have a MSRV (#135) we should add a note here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+This document has a few pointers and guidelines to help you get started.
+It is not perfect, and there will always be exceptions to the rules described
+here, but by following the instructions below you should have a much easier time
+getting your work merged with the upstream project.
+
+To have a welcoming and inclusive project, libseccomp-rs uses the Rust project's
+[Code of Conduct][code-of-conduct]. All contributors are expected to follow it.
+
+[code-of-conduct]: https://www.rust-lang.org/conduct.html
+
```
Comment (rusty-snake) [inline CONTRIBUTING.md:13]:
```suggestion
## Questions

You can use [GitHub Discussions] to ask questions (Category Q&A).
As for issues, you should first search if you question might be already asked (and answered).

[GitHub Discussions]: https://github.com/libseccomp-rs/libseccomp-rs/discussions
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+We're really glad you're interested in contributing to libseccomp-rs!
+This document has a few pointers and guidelines to help you get started.
+It is not perfect, and there will always be exceptions to the rules described
+here, but by following the instructions below you should have a much easier time
+getting your work merged with the upstream project.
+
+To have a welcoming and inclusive project, libseccomp-rs uses the Rust project's
+[Code of Conduct][code-of-conduct]. All contributors are expected to follow it.
+
+[code-of-conduct]: https://www.rust-lang.org/conduct.html
```
[the remark this answers is not in the document]
Comment (ManaSugi) [reply CONTRIBUTING.md]: Ok, I fixed it.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
+# Contributing to libseccomp-rs
+
+We're really glad you're interested in contributing to libseccomp-rs!
+This document has a few pointers and guidelines to help you get started.
+It is not perfect, and there will always be exceptions to the rules described
+here, but by following the instructions below you should have a much easier time
+getting your work merged with the upstream project.
+
+To have a welcoming and inclusive project, libseccomp-rs uses the Rust project's
+[Code of Conduct][code-of-conduct]. All contributors are expected to follow it.
```
[the remark this answers is not in the document]
Comment (ManaSugi) [reply CONTRIBUTING.md]: Sure, I removed it.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+### Bug Reports
+
+Before submitting a new bug report, please search existing issues to see if
+there's something related. If not, just [open a new issue][new-issue]!
+
+The more information you can give in your issue, the easier it is to figure out
+how to fix it. For libseccomp-rs, this will likely include the
+[upstream libseccomp][upstream-libseccomp] version, the Linux kernel version ,and
+the architecture. In addition, it is incredibly helpful if you can describe/include
```
[the remark this answers is not in the document]
Comment (ManaSugi) [reply CONTRIBUTING.md]: Good. Thank you.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ManaSugi) [reply CONTRIBUTING.md:43]: Good.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ManaSugi) [reply CONTRIBUTING.md:16]: I added the section. Thank you for the suggestions.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+a reproducer for the problem in the description as well as instructions on how to
```
[the remark this answers is not in the document]
Comment (ManaSugi) [reply CONTRIBUTING.md]: I fixed it.

**Comment by ManaSugi** [comment]
I should replace the "open a new issue" link with the bug template link after #145 is merged.

**Review by rusty-snake — APPROVED** [review]

**Comment by ManaSugi** [comment]
@rusty-snake Thank you for the review!

**Pull request merged**